app-config-exec/README.md

```diff
@@ -0,0 +1,96 @@
+## App Config Exec Plugin
```

Comment:
A decent example - using `hostname` command for a "built by" metadata

Reply:
Excellent example, I'll replace the first example with this.